chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -233,45 +359,104 @@ class CurrentUser(
             callback: (Result<Int, Error>) -> Unit
     ) = makeCallback({ syncCurrentUser.sendMessage(roomId, messageText, attachment) }, callback)
 
+    /**
+     * Send a [messageText] to the [roomId]. The [callback] will be called with a [Result] when the
+     * operation is complete. If the operation was successful you will receive an [Int] which
+     * contains the message id, or an [Error] informing you of what went wrong.
+     */
```

Comment:
Perhaps we should be more explicit that this is a convenience method, and so spell out what it is equivalent to - a message with a single part of type `text/plain` and content of `messageText`, and an `@see sendMultipartMessage`

Reply:
Good point. I've written it like:
```
This is a convenience method which will use the [sendMultipartMessage] which will send a single
     * `NewPart.Inline([messageText])` to the [roomId].
``` ?